src/dlgProfilePreferences.cpp

```diff
@@ -157,11 +157,19 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pHost)
                                         .arg(tr("<p>Select the only or the primary font used (depending on <i>Only use symbols "
                                                 "(glyphs) from chosen font</i> setting) to produce the 2D mapper room symbols.</p>")));
     checkBox_isOnlyMapSymbolFontToBeUsed->setToolTip(QStringLiteral("<html><head/><body>%1</body></html>")
-                                        .arg("<p>Using a single font is likely to produce a more consistent style but may "
-                                             "cause the <i>font replacement character</i> '<b>�</b>' to show if the font "
-                                             "does not have a needed glyph (a font's individual character/symbol) to represent "
-                                             "the grapheme (what is to be represented).  Clearing this checkbox will allow "
-                                             "the best alternative glyph from another font to be used to draw that grapheme.</p>"));
+                                                     .arg(tr("<p>Using a single font is likely to produce a more consistent style but may "
+                                                             "cause the <i>font replacement character</i> '<b>�</b>' to show if the font "
+                                                             "does not have a needed glyph (a font's individual character/symbol) to represent "
+                                                             "the grapheme (what is to be represented).  Clearing this checkbox will allow "
+                                                             "the best alternative glyph from another font to be used to draw that grapheme.</p>")));
+    checkBox_runAllKeyBindings->setToolTip(QStringLiteral("<html><head/><body>%1</body></html>")
+                                           .arg(tr("<p>If <b>not</b> checked Mudlet will utilise only the first matching combination of key "
+                                                   "and modifier even if more than one of them is set to be active. This means that a "
+                                                   "temporary key binding (not visible in the Editor) created by a script or "
+                                                   "package may be used in preference to a permanent one that is shown and "
+                                                   "is set to be active. If checked then all matching keybinding will be run.</p>"
+                                                   "<p><i>It is recommended to not enable this option if you need to maintain compatibility "
+                                                   "with scripts or packages for Mudlet versions prior to <b>3.9.0</b>.</i></p>")));
```

Comment:
I agree and especially on the length (slysven would know), most application tooltips are much shorter.

Reply:
For consistency I'd would change the first usage to "Key-binding (combination of key and modifier)" to remind the user of that type and then use "Key-binding" for the other usages. Is that acceptable?

Also, thinking about this a bit more, I'd avoid using just "Key" as the identifier for this functionality because it is hard to search for as it is also associated, of course, with lua tables and other look-up situations.

Reply:
Just "keybinding" would work.

Reply:
Revised to use "keybinding", with the first one followed by "(combination of key and modifiers)".